truss-build/src/truss/build/commands.py

```diff
@@ -0,0 +1,93 @@
+from abc import abstractmethod
+import shlex
+
+
+# TODO(bola): Support secrets
+class Command:
+    @abstractmethod
+    def serialize(self):
+        pass
+
+
+class FromCommand(Command):
+    def __init__(self, image, tag=None, AS=None):
+        self.image = image
+        self.tag = tag
+        self._as = AS
+
+    def serialize(self):
```

Comment:
Add `str` return type.

When I read "serialize" I usually assume it's bytes.

Maybe even consider renaming this method to `to_str` or `as_str` so it's immediately deductable from the name. Since this gets later called as `write_text(image.serialize())` it might also make sense `as_text`.